application/common/core.cpp

```diff
@@ -557,55 +598,58 @@ main(int argc, char *argv[])
     // Set event callback for events that cannot wait
     SDL_SetEventFilter(sdlPriorityEvents, NULL);
 
-    int stencilSize = 1;
-    ret = SDL_GL_SetAttribute(SDL_GL_STENCIL_SIZE, stencilSize);
-    lmAssert(ret == 0, "SDL Error: %s", SDL_GetError());
+    if (!gLoomHeadless) {
+
+        int stencilSize = 1;
+        ret = SDL_GL_SetAttribute(SDL_GL_STENCIL_SIZE, stencilSize);
+        lmAssert(ret == 0, "SDL Error: %s", SDL_GetError());
```

Comment:
Probably deserves a better message that won't make us insane if someone reports it?